### ISSUE TITLE
unconfirmed bug in python code

### ISSUE THREAD

**Issue by adalisan** [jbloomlab/dms_tools2#59]
I was reading through the code, and I think this is likely a bug.

https://github.com/jbloomlab/dms_tools2/blob/234670ca480eeaf04a98335c1cf3ff088ab9ab92/scripts/dms2_bcsubamp#L354
```
if r2trim is None:
     r2trimconsensus = consensus['R2'][ : r2trim]
```
if this is true, `None` would be an index for the string. I think the right side was  meant to be `consensus['R2']` just like for  r1trim. Probably not an issue in a practical sense.

**Comment by jbloom** [comment]
Yes, you are correct---that is a bug. Feel free to submit a pull request if you'd like, or I can try to fix at some point.
